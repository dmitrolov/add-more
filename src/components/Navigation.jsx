import React from 'react';

export const Navigation = () => {
    return (
        <>
            <div>test text</div>
        </>
    )
}