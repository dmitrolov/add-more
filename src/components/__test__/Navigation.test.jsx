import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';

import { Navigation } from '../Navigation';

configure({ adapter: new Adapter() });
let defaultProps;
let wrapper;

const setup = (props = {}) => shallow(<Navigation {...defaultProps} {...props} />)

beforeEach(() => {
    wrapper = setup()
});

afterEach(() => {
    jest.clearAllMocks()
})

describe('Navigation', () => {
    it('should match snapshot', () => {
        console.log(wrapper.debug())
        expect(wrapper).toMatchSnapshot()
    })
})