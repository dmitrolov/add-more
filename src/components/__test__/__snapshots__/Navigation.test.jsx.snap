// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation should match snapshot 1`] = `
<Fragment>
  <div>
    test text
  </div>
</Fragment>
`;
